frontend_lib/src/TracimTableLib/TracimTable/TracimTable.jsx

```diff
@@ -0,0 +1,170 @@
+          onChange={e => {
+            const newFilter = e.target.value
+            table.setGlobalFilter(newFilter)
+          }}
+          value={table.getState().globalFilter || ''}
+          placeholder={props.filterPlaceholder}
+        />
+      )}
+      {rows.length > 0
+        ? (
+          <>
+            {!props.noHeader && (
+              <div className='tracimTable__header'>
+                {table.getHeaderGroups().map(headerGroup => (
+                  <div
+                    className='tracimTable__header__row'
+                    key={headerGroup.id}
+                  >
+                    {headerGroup.headers.map(header => (
+                      <div
+                        onClick={props.sortable && header.column.getToggleSortingHandler()}
+                        className={classnames('tracimTable__header__row__cell',
+                          header.column.columnDef.className,
+                          { tracimTable__styles__clickable: props.sortable }
+                        )}
+                        key={header.id}
+                      >
+                        {header.isPlaceholder
+                          ? null
+                          : (
+                            <>
+                              {flexRender(
+                                header.column.columnDef.header,
+                                {
+                                  ...header.getContext()
+                                }
+                              )}
+                              {props.sortable && (
+                                <Icon
+                                  icon={{
+                                    asc: 'fas fa-sort-amount-down-alt',
+                                    desc: 'fas fa-sort-amount-up-alt'
+                                  }[header.column.getIsSorted()] || null}
+                                  customClass={classnames('titleListHeader__icon', { tracimTable__styles__hide: !header.column.getIsSorted() })}
+                                  title={header.column.columnDef.tooltip || ''}
+                                />
+                              )}
+                            </>
+                          )}
+                      </div>
+                    ))}
+                  </div>
+                ))}
+              </div>
+            )}
+            <div className={classnames('tracimTable__body', { tracimTable__body__colored: props.colored })}>
+              {rows.map(row => {
+                const Line = props.lineComponent || GenericTracimTableLine
+                return (
+                  <Line key={`${row.id}-line`} rowData={row} customRowClass={props.customRowClass} />
+                )
+              })}
+            </div>
+          </>
+        )
+        : (
+          <EmptyListMessage>
+            {
+              props.data.length <= 0
+                ? props.emptyMessage
+                : 'Nothing matches you filter'
```

Comment:
Shouldn't this be translated or <EmptyListMessage> translate it?

Reply:
This should be translated. I indeed forgot the translation